### PR TITLE
Bookapi

### PR DESCRIPTION
**Pull request by ceifeirocv**
Progress in the API 
Created an API route for book search, currently utilizing the Google API.
Developed an API route to search for book prices, specifically using the Google Books API.
Acknowledged that Amazon's API is challenging to work with.
Open to exploring other bookstores for price comparisons.

### CODE REVIEW

File: bookfinder/src/app/api/v1/book/route.ts
Comment (EmmanuelOloke) [inline bookfinder/src/app/api/v1/book/route.ts:1]: Great work man. A bit of documentation might be needed for ease of usage later on though, but the implementation looks solid to me.

File: bookfinder/src/app/api/v1/price/route.ts
Comment (EmmanuelOloke) [inline bookfinder/src/app/api/v1/price/route.ts:1]: Nice. Looks good to me

**Review by EmmanuelOloke — DISMISSED** [review]
Later one API usage may need to be documented in README.md to improve the developer experience. But the implementations look good to me for now. I approve.

**Comment by ceifeirocv** [comment]
I will try to add documentation later,

**Comment by EmmanuelOloke** [comment]
> 

I can assist with that later on if necessary and you don't mind. Nice work!

**Review by yildiztugba — DISMISSED** [review]

**Review by Pio-js — DISMISSED** [review]

**Review by yildiztugba — APPROVED** [review]

**Review by Pio-js — APPROVED** [review]

**Pull request merged**